app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_user!
+      def index
+        user = User.find(params[:user_id])
+        render_paginated user.rents, each_serializer: RentSerializer
+      end
+
+      def create
+        user = User.find(params[:user_id])
+        new_rent = user.rents.new(rent_params)
+        if new_rent.save
+          RentMailer.new_rent_created(new_rent).deliver_later
```

Comment:
When sending parameters to an async worker or mailer objects are serialized by redis. Instead of sending them use their id

Reply:
It's good to know. Done!.